.eslintrc.js

```diff
@@ -83,7 +83,7 @@ module.exports = {
       'requireReturnForObjectLiteral': false
     }],
 
-    'arrow-parens': ['error', 'always'],
+    // 'arrow-parens': ['error', 'always'],
```

Comment:
I commented out this rule because sometimes arrow functions if they have only one parameter don't need parens

Reply:
It's fine to enable. I want to be consistent and always use parens.

Reply:
I have added back this rule